workshops/Tkinter/README.md

```diff
@@ -0,0 +1,188 @@
+
+```python
+window.mainloop()
+```
+
+window.mainloop() tells Python to run the Tkinter event loop. ... mainloop() at the end of a program in a Python file, then the Tkinter application will never run, and nothing will be displayed.
+
+###### Now the code will look like this.
+
+```python
+import tkinter as tk
+window = tk.Tk()
+window.minsize(800, 560)
+window.title("Simple UI")
+window.config(bg="black")
+tk.Label(window, font=("Helvetica", 60, "bold"), text="QUOTES HUB", bg="black",fg="white").pack()
+tk.Button(window,command=fun1,text="Get In",bg="aqua",fg="black",width=20).place(x=315,y=300)
+window.mainloop()
+```
+
+As output, this will show an error that the function fun1 is not declared. So let us make the button work. You need to put this code above the window declaration then only it will work.
+
+```python
+def fun1():
+  F1=tk.Frame(window,bg="light salmon",width=800,height=560).place(x=0,y=0)
+  tk.Label(F1,text="Creativity \nis \nintelligence \nhaving \nFun.",font=("Helvetica", 30, "bold"),bg="light salmon",fg="black").pack()
+  tk.Button(F1,command=fun2,text="Next",bg="aqua",fg="black",width=20).place(x=612,y=530)
+```
+
+In this function, we are showing up a new page in which we display a quote and a next button. As pages, we are using the Frame function. A-Frame widget is used to organize a group of widgets. It acts like a container that can be used to hold the other widgets. The rectangular areas of the screen are used to organize the widgets to the python application. Then in that frame, we display the quote at the center using the label function. Then we create a next button and give it a command fun2. Now let us create that function.
+
+```python
+def fun2():
+   F2=tk.Frame(window,bg="light salmon",width=800,height=560).place(x=0,y=0)
+   tk.Label(F2,text="If the plan \ndoesn’t work, \nchange the plan,\n but never the \ngoal.",font=("Helvetica", 30, "bold"),bg="light salmon",fg="black").place(x=200,y=100)
+   tk.Button(F2,command=fun3,text="Next",bg="aqua",fg="black",width=20).place(x=612,y=530)
+```
+
+This function will be the next page with another quote. If you don't understand just go to the live demo and feel the button workflow [here](https://repl.it/@bezlin/graphics#main.py). Ok in this function same as before we are creating a frame and in it, we will displaying another quote and a next button. Like that we have created 5 pages. On the last page, you can avoid the next button.
+
+You need to create more functions according to how many pages you want that's it pretty cool.
```

Comment:
can we not create a more efficient array using an array of quotes and then select one at random? would be much nicer imo

Reply:
ok I will do that